src/prism.c

```diff
@@ -5687,12 +5687,16 @@ pm_parser_scope_push_transparent(pm_parser_t *parser) {
 static int
 pm_parser_local_depth(pm_parser_t *parser, pm_token_t *token) {
     pm_constant_id_t constant_id = pm_parser_constant_id_token(parser, token);
+    pm_constant_id_t ellipsis_id = token->type == PM_TOKEN_UAMPERSAND ? pm_parser_constant_id_constant(parser, "...", 3) : 0;
```

Comment:
What do you think about passing this as a parameter so the logic of this function stays the same but the caller sites can change its behavior?

Reply:
Indeed, that sounds good.